app/views/schools/cohorts/programme_choice_no_early_career_teachers.html.erb

```diff
@@ -5,7 +5,7 @@
   <div class="govuk-grid-column-two-thirds">
     <span class="govuk-caption-l"><%= @cohort.academic_year %> cohort</span>
     <h1 class="govuk-heading-xl govuk-!-margin-top-2">You are not expecting any early career teachers this year</h1>
-    <p class="govuk-body-l">
+    <p class="govuk-body">
       If you employ early career teachers later this year, you can <%= govuk_link_to "nominate an induction tutor", resend_email_request_nomination_invite_path %> to set up your programme.
```

Comment:
I'm not sure this is true currently, is it?

Reply:
We've designed a journey under this ticket: https://dfedigital.atlassian.net/browse/CPDRP-906?atlOrigin=eyJpIjoiZTczYTI0MzNjNWUxNGQ1NWI0YTIxYmNiZmJiYWJkYTgiLCJwIjoiaiJ9

Reply:
But that's unlikely to go live before this - and either way this will be the wrong link. Nominate an induction tutor is never the correct action here

Reply:
Should this be a 'contact us' link instead then?

Reply:
Yes, you can just `<%= render MailToSupportComponent.new("contact us") %>`